openshift/templates/deploy.dc.yaml

```diff
@@ -85,6 +91,10 @@ objects:
               imagePullPolicy: Always
               name: ${NAME}
               env:
+                - name: PORT
+                  value: ${PORT}
+                - name: WORKERS_PER_CORE
```

Comment:
uvicorn-gunicorn-fastapi uses uvicorn-gunicorn-docker which only exposes 80 (see:
https://github.com/tiangolo/uvicorn-gunicorn-docker/blob/master/docker-images/python3.8.dockerfile ) - so we have to export 8080 since nothing else exposes is.

Reply:
Ah okay this line is about setting a running port for the api. Gotcha thx!